.gitattributes

```
@@ -1,2 +1,11 @@
 #Set default behavior to use Unix line endings so Docker on Windows doesn't fail
 * text eol=lf
+
+# Don't normalize images and other binary files
+# more info: https://www.git-scm.com/docs/gitattributes
+*.ai -text
+*.png -text
+*.ttf -text
+*.pdf -text
+*.jpg -text
+*.ico -text
```

I'm perfectly happy to add .txt to the list here

```suggestion
*.ico -text
*.txt -text
```

---

oh, i was thinking we _shouldn't_ add txt, because we would want those line breaks to be converted. just that one file has to be manually committed (i can do that on a separate commit)

---

the reason these other files were showing up is because the magic numbers (file headers) for some files include CRLF for legacy reasons - they definitely should not be converted; anything plain text/code should be (is my understanding)